_data/versions.yml

```diff
@@ -1186,6 +1186,48 @@ v3.2:
       url: https://github.com/projectcalico/pod2daemon/releases/tag/v3.2.0
 
 v3.1:
+- title: v3.1.7
+  components:
+    calico/node:
+      version: v3.1.7
+      url: https://github.com/projectcalico/calico/releases/tag/v3.1.7
+    calicoctl:
+      version: v3.1.7
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v3.1.7
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v3.1.7/calicoctl
+    calico/cni:
+      version: v3.1.7
+      url: https://github.com/projectcalico/cni-plugin/releases/tag/v3.1.7
+      download_calico_url: https://github.com/projectcalico/cni-plugin/releases/download/v3.1.7/calico
```

Comment:
These download URLs are correct

Reply:
I just checked and the URL looks right to me - https://github.com/projectcalico/cni-plugin/releases/tag/v3.1.7